### ISSUE TITLE
Going directly to a bitzer from start...

### ISSUE THREAD

**Issue by Nsane-FFXI** [AkadenTK/superwarp#57]
You made it so it has to detect the KI, but there an inventory item KI so it has a load time on them upon zoning, their permanent items can u make it so it stores the permanent KI's to a character some how instead of getting the message "You do not have any of the Ra'Kaznar Plates".

Stupid issue, it usually takes a few more second until they load but figured I ask. Great job though, Amazing.

**Comment by AkadenTK** [comment]
I would prefer not to keep storage, since it could lead to edge-cases. You *can* drop those KIs IIRC. In that case, it could lead to a situation (a rare one, surely) where Superwarp is not loaded and the KI is dropped, then Superwarp is loaded and an attempt is made to use the bitzer without the KI state being updated. 

There's some stuff to be done to mitigate that, and it might not even be a real concern. But what it can't mitigate is falsifying the records. I'm not certain what the NPC will do if you try to port without the KI. It might be very bad for you like flagging you for a ban or even an immediate boot from the zone, or it might let you cheat by skipping the KI. And either way, I really don't want to let that happen.

I'm more curious if the menu "knows" what KIs you have before your inventory loads. If that's the case, then we can probably use the menu itself instead of having to wait for the KIs to load.

**Comment by Nsane-FFXI** [comment]
I'm guessing the menu knows or what ever your talking about, only because I know I can do it manually right off the start. I think if you drop them those items they come back into your inventory instantly. Like dropping the Obsidian wing.

**Comment by staticvoid0** [comment]
Yeah this is the only thing left to do, I just haven't had time, nor a character without device access to develop a fix for this particular issue and I understand as much as anyone about precious seconds in sortie.    
![no problem](https://github.com/user-attachments/assets/2d1cb311-628e-4c86-a0b7-67237e6fc040)

**Comment by Nsane-FFXI** [comment]
Thanks for the response.

**Comment by staticvoid0** [comment]
I had forgotten about this but I fixed that. All KI checks are nixed, it reads unlock from the menu now with version 1.1.1 
Instant //sw a,b,c,d  when you come in now. cheers
